app/src/test/java/com/aws/amazonlocation/utils/generalutils/GeneralUtilsValidateLatLngTest.kt

```diff
@@ -20,13 +20,13 @@ class GeneralUtilsValidateLatLngTest : BaseTest() {
 
     @Test
     fun testValidateLatLngInvalidLat() {
-        val result = validateLatLng(LAT_LNG_VALID_LNG_STRING)
+        val result = validateLatLng(LAT_LNG_VALID_LAT_STRING)
         Assert.assertTrue(TEST_FAILED_DUE_TO_INVALID_LATITUDE, result == null)
     }
 
     @Test
     fun testValidateLatLngInvalidLng() {
-        val result = validateLatLng(LAT_LNG_VALID_LAT_STRING)
+        val result = validateLatLng(LAT_LNG_VALID_LNG_STRING)
         Assert.assertTrue(TEST_FAILED_DUE_TO_INVALID_LONGITUDE, result == null)
     }
```

Comment:
Why are these tests passing? It says it's passing in a "VALID_LAT_STRING" but is then asserting that the test failed due to invalid latitude. (Same situation with longitude)

These strings seem reversed, but it's also disturbing that this isn't causing test failures. Are we sure the validation works?

Reply:
Yes, it seems that the method names do not match the test cases. I have updated the method and string names. We are testing whether the latitude is invalid, so the validateLatLng method will return a null value, and the same applies to longitude.